### DISCUSSION TITLE
v1.112.1 first build

### DISCUSSION THREAD

**Discussion by 4v3ngR** [4v3ngR/immich-native-macos#9] [Announcements]
ALPHA quality software - use protection.

This is an unofficial macOS installer for immich version v1.112.1

<hr /><em>This discussion was created from the release <a href='https://github.com/4v3ngR/immich-native-macos/releases/tag/v1.112.1-a'>v1.112.1 first build</a>.</em>

**Comment by Al3xESP** [comment]
I load it using the package loader and it seems to be working except when I log in the web app it show pictures only as blank thumbnails.  video will preview if I hold mouse over it.  If I click on any picture/video it just shows a larger blank thumbnail.   Running on Mac OS Montery, latest update.  

![immich only showing thumbnails web app](https://github.com/user-attachments/assets/72c526e9-2b45-4d9b-8927-8aecd37d92a6)
